### DISCUSSION TITLE
Unexpected rounding of numbers with lot of fraction digits in grid

### DISCUSSION THREAD

**Discussion by michal-bartak** [dbeaver/dbeaver#15930] [General]
#### System information: 
- Fedora 35
- DBeaver version: 21.3.4
- Additional extensions: N/A

#### Connection specification:
- Postgresql 12/13
- Postgres
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
It seems that grid unexpectedly rounds numbers if the number of fraction digits exceeds some limit.
Here is an example:

`SELECT 0.009999999, 0.00999999999999`

Result in text view:
![image](https://user-images.githubusercontent.com/7868445/154492133-264edc5c-902b-4b21-8c45-f33288cd7963.png)

Result in grid:
![image](https://user-images.githubusercontent.com/7868445/154492201-97f3c13a-d96f-49b8-a29e-dd482f393c3b.png)

#### Steps to reproduce, if exist:
See above

#### Include any warning/errors/backtraces from the logs
N/A

**Comment by uslss** [comment]
@michalk-k number of fraction digits could be changed in connection or global preferences: Editors - Data Editor - Data Formats
![image](https://user-images.githubusercontent.com/31996417/154917087-6935014c-60c2-49b9-ab45-de0746684b08.png)

**Comment by michal-bartak** [comment]
Thank you. It works and I'm happy with it.
But still have mixed feelings about the default settings which skews data significantly without warning.
Also found no option to just truncate to maximum displayed number of digits. Only rounding down works with my example but obviously it must not work in other cases.

BTW Postgres documentation says:

> up to 131072 digits before the decimal point; up to 16383 digits after the decimal points

DBeaver settings have accepted this number :) Hope it's expected and will remain working. 

Anyway, since DBeaver knows the real number, maybe the rounded value should be somehow marked, for example by exclamation icon or red color etc. Otherwise a user may never figure out he is looking at wrong numbers.
